tests/modules/nf-core/proteinortho/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PROTEINORTHO } from '../../../../modules/nf-core/proteinortho/main.nf' addParams( options: [args: '-e=10'] )
```

Comment:
Oh, addParams hasn't been around for a long time. This should be via the nextflow.config file if your need it

Reply:
changed